@anonklub/proof/package.json

```diff
@@ -19,7 +19,6 @@
     "privacy"
   ],
   "license": "MIT",
-  "files": ["dist"],
```

Comment:
we must keep this.
Otherwise the ui build step will fail because it can't find the `@anonklub/proof` module. Especially in GH actions.

cc @0xisk 
https://github.com/anonklub/anonklub/actions/runs/7697807334/job/20975621133?pr=308#step:12:2324